software/deploy/deploy.go

```diff
@@ -80,17 +80,17 @@ func Airflow(houstonClient houston.ClientInterface, path, deploymentID, wsID, by
 	}
 	deployments, err := houston.Call(houstonClient.ListDeployments)(request)
 	if err != nil {
-		return err
+		return deploymentID, err
 	}
 
 	c, err := config.GetCurrentContext()
 	if err != nil {
-		return err
+		return deploymentID, err
 	}
 
 	cloudDomain := c.Domain
 	if cloudDomain == "" {
-		return errNoDomainSet
+		return deploymentID, errNoDomainSet
```

Comment:
Finding it tricky to cover this line. Since this line is already uncovered by test and it's not a new line, I am leaving it as it is.